validator/sawtooth_validator/gossip/gossip_handlers.py

```diff
@@ -220,11 +220,26 @@ def _has_open_requests(self, batch_id):
         return self._responder.get_request(batch_id)
 
 
+class GossipConsensusHandler(Handler):
+    def __init__(self, gossip, notifier):
+        self._gossip = gossip
+        self._notifier = notifier
+
+    def handle(self, connection_id, message_content):
+        obj, tag, _ = message_content
+
+        if tag == GossipMessage.CONSENSUS:
+            self._notifier.notify_peer_message(
+                message=obj,
+                sender_id=bytes.fromhex(
```

Comment:
[Open Ended Question]: I don't know if these are thought through, but here are concerns around consensus peer messages
1. It should be possible to set priority of consensus peer messages through consensus engine. Which would imply apart from handling gossip messages with priority at dispatcher queue of same validator, it should be possible for other validators to process these messages faster.
2. Ability to have dynamic peering of validators, yet have consensus engine be able to talk to a peer consensus engine not directly connected. Would it be feasible to add peer info in message itself and then rely on gossip network to carry consensus peer message to destined consensus engine.

Reply:
Those are very good questions. Just to clarify, for (1): do you mean that the consensus engine should be able to decide how high of a priority the consensus messages are? And that this means consensus messages should be separate from Gossip messages so that their priority is independent? And for (2): are you suggesting that it would be useful for some networks using a consensus engine that doesn't require full peering (or maybe full peering isn't possible) to have the broadcast functionality to relay messages to nodes that may not be directly connected? And maybe they can do it intelligently (not broadcast to everyone, just send directly to the intended peer)?

Reply:
1. Yes, consensus messages should be independent of other gossip messages validators exchange. This way few consensus messages can be handled with priority where immediate action is required.
2. Yes.

Reply:
These are good ideas that are worth exploring. We're already talking about (and considering an RFC for) some way for the consensus engine to give the validator hints about how to handle messaging. We'll be sure to include these ideas!